File: functions/notes.py
# Johann hines, Functions notes in python 

#function is ana action stored in a key word 
#number = int(input("can i get a number:\n"))
#def add(numOne, numTwo):#paramaters go in the parenthasis 
   # numOne = int(("Give me a numberp:\n"))
   # numTwo = int(("Give me a numberp:\n"))
    
  #  print(numOne+numTwo)
 #add(int(input("tell me a number:\n"))
    #add(12+34)#arguments ar egiven when the function is called and must match the number of parameters
    #add(13+12)
    #add(10+10)
def user(word):
 return input(f"tell me a {word}:\n")
name = user("name")
verb = user("verb")
place = user("place")
print(f"{name} was {verb} amd somehow got to {place}.")

File: functions/notes.c
//johann hines, expressions notes
#include <stdio.h>

void add(int numOne, int numTwo){
    printf("%d\n", numOne + numTwo);
}
char input(char type[20]){
 char answer[50];
 printf("please give me a %s;\n",type);
 scanf("%s", answer);
 return answer;
}


int main(void){
   // printf("hello world\n");
   // add(4,9);
    //add(5,10);
   // add(1,99);
   // add(5,7);
    input("name");
    input("verb");
    input("place");
    printf("%s was %s untill they somehow reached %s", input("name"), input("verb"), input("place"));
    return 0; 
}